content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -7,23 +7,34 @@ weight: 40
 ---
 
 {{% capture overview %}}
+{{< feature-state for_k8s_version="v1.1" state="beta" >}}
 {{< glossary_definition term_id="ingress" length="all" >}}
 {{% /capture %}}
 
 {{% capture body %}}
+
 ## Terminology
 
 For the sake of clarity, this guide defines the following terms:
 
-* Node: A single virtual or physical machine in a Kubernetes cluster.
-* Cluster: A group of nodes firewalled from the internet, that are the primary compute resources managed by Kubernetes.
-* Edge router: A router that enforces the firewall policy for your cluster. This could be a gateway managed by a cloud provider or a physical piece of hardware.
-* Cluster network: A set of links, logical or physical, that facilitate communication within a cluster according to the [Kubernetes networking model](/docs/concepts/cluster-administration/networking/).
-* Service: A Kubernetes [Service](/docs/concepts/services-networking/service/) that identifies a set of pods using label selectors. Unless mentioned otherwise, Services are assumed to have virtual IPs only routable within the cluster network.
+Node
+: A worker machine in Kubernetes, part of a cluster.
+
+Cluster
+: A set of Nodes that run containerized applications managed by Kubernetes. For this example, and in most common Kubernetes deployments, nodes in the cluster are not part of the public internet.
```

Comment:
nit: a Cluster is a K8s control plane (aka masters) + workers/nodes/compute. Maybe there will be some value to add this info in the paragraph

Reply:
Maybe it'd be better to adopt the glossary definition for each term defined there. It doesn't seem a problem to use the generally accepted definition of a Kubernetes cluster, for example.

I'll think about this.

Reply:
@sftim ignore my comment for now as there is a wider discussion on K8s mailer which involves sig-cluster-lifecycle/ architecture to address/ agree on the definition - so far no agreement